app/index.html

```diff
@@ -185,8 +187,8 @@ <h1 ng-bind="globalPage.title"></h1>
   </header>
   <section id="frontpagetitle">
     <div class="container">
-      <div class="row">
-        <div class="col-xs-10">
+      <div class="row spacing-above-md spacing-below-md">
+        <div class="col-sm-10 stacked-spacing-col-sm">
           <a ng-href="#/{{registerMode || 'register'}}/{{globalPage.confId}}/page/" ng-if="globalPage.confId">
```

Comment:
A hyperlink underline is added when hovering over the event name.  That should be removed.